src/test/resources/sample.servlet/testAppendServerEnvWithEnvProps.gradle

```diff
@@ -0,0 +1,94 @@
+/*
+	This test checks whether the application was successfully installed without the version number in the package
+	when deploy is called and stripVersion is set to true.
+*/
+group = 'liberty.gradle'
+version = '1'
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            name = 'Sonatype Nexus Snapshots'
+            url = 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
+    }
+    dependencies {
+        classpath "io.openliberty.tools:liberty-gradle-plugin:$lgpVersion"
+    }
+}
+
+apply plugin: 'war'
+apply plugin: 'liberty'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+compileJava.options.encoding = 'UTF-8'
+
+ext {
+    // Liberty server properties
+    wlpServerName = 'LibertyProjectServer'
+    serverDirectory = "${project.buildDir}/wlp/usr/servers/${wlpServerName}"
+    testServerHttpPort = 9080
+    testServerHttpsPort = 9443
+
+    // This is set in the ibm-web-ext.xml file
+    warContext = 'myLibertyApp'
+
+}
+
+liberty {
+    server{
+        serverXmlFile = file("src/main/liberty/config/server-apps-test.xml")
+        name = wlpServerName
+        deploy {
+            apps = [war]
+            copyLibsDirectory = file("${project.buildDir}/libs")
+        }
+        appendServerEnv = true
+        serverEnvFile = file("src/resources/server.env")
+        configDirectory = file('src/resources/config')
+        env = ['TEST_PROP_2':'white', 'CONFIG_SERVER_ENV_PROPS':'TEST']
```

Comment:
To prove that the env props take precedence over the serverEnvFile, I would like the same property to appear in both with a different value. The test case as it stands proves that all four sources got merged, and that env props take precedence over configDir and serverEnvFile takes precedence over configDir.